src/couch_replicator_httpc.erl

```diff
@@ -257,22 +270,25 @@ discard_message(ReqId, Worker, Count) ->
         exit({timeout, ibrowse_stream_cleanup})
     end.
 
-
```

Comment:
I think I got this one. But I am not sure. I don't see the change on that line in latest diff